### PR TITLE
add supported environments in readme

### PR DESCRIPTION
**Pull request by j-zhong-ms**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,15 +8,19 @@ Welcome to the home page of `Azure Monitor OpenTelemetry Profiler for .NET (PREV
 
 ### Step 0: Prerequisites
 
-- **.NET 8 SDK**: Install the latest .NET Core SDK from [here](https://dotnet.microsoft.com/download/dotnet).
+- **.NET 6.0 or later**: Install the latest .NET SDK from [here](https://dotnet.microsoft.com/download/dotnet).
+- **Supported Environments**:
 
-### Step 1: Create an ASP.NET Core Application
+  - Windows
```
Comment (xiaomi7732) [inline README.md:14]: Is Linux supported or not?

File: README.md [hunk 2]
```diff
@@ -8,15 +8,19 @@ Welcome to the home page of `Azure Monitor OpenTelemetry Profiler for .NET (PREV
...
+  - Containerized apps in Windows or Linux
+  - Azure App Service
+  - Azure Kubernetes Service (AKS)
+
```
Comment (xiaomi7732) [inline README.md:18]: While you are here, do you mind add a pre-req of application insights resource?

File: README.md [hunk 3]
```diff
@@ -8,15 +8,19 @@ Welcome to the home page of `Azure Monitor OpenTelemetry Profiler for .NET (PREV
...
+### Step 1: Create a .NET Application
 
 If you don't have an app already, you can create a new web API project using the following command:
 
-```sh
```
Comment (xiaomi7732) [inline README.md:17]: Are you removing the code section tags, why? ("```sh")

**Review by xiaomi7732 — APPROVED** [review]
Generally looks good to me.

File: README.md (hunk 2 above)
Comment (j-zhong-ms) [reply README.md:19]: Sure, added

File: README.md (hunk 3 above)
Comment (j-zhong-ms) [reply README.md]: Good catch! Added it back

File: README.md (hunk 1 above)
Comment (j-zhong-ms) [reply README.md:14]: Added to the list as well

**Review by xiaomi7732 — APPROVED** [review]

**Pull request merged**